proto/trogonai/grpc_nats_micro/v1/echo.proto

```
@@ -0,0 +1,45 @@
+edition = "2024";
+
+// Reference service for the grpc-nats-micro binding (ADR 0016). It is the
+// conformance fixture the binding's integration tests drive, and a worked
+// example of a protobuf service annotated for NATS micro. It carries no domain
+// meaning.
+package trogonai.grpc_nats_micro.v1;
+
+import "google/rpc/code.proto";
```

Buf (1.72.0)

[error] imported file does not exist

(COMPILE)